server/test/controllers/admin/AdminApplicationControllerTest.java

```diff
@@ -164,6 +165,35 @@ public void index() throws Exception {
     assertThat(result.status()).isEqualTo(OK);
   }
 
+  @Test
+  public void index_showsBulkStatusViewWhenFlagOn() throws Exception {
+    Request request = fakeRequest();
+    when(settingsManifestMock.getBulkStatusUpdateEnabled(request)).thenReturn(true);
+    controller = makeNoOpProfileController(/* adminAccount= */ Optional.empty());
+
+    ProgramModel program = ProgramBuilder.newActiveProgram().build();
+    ApplicantModel applicant = resourceCreator.insertApplicantWithAccount();
+    applicant.refresh();
+    ApplicationModel application =
+        ApplicationModel.create(applicant, program, LifecycleStage.ACTIVE).setSubmitTimeToNow();
+    application.refresh();
+    Result result =
+        controller.index(
+            fakeRequest(),
+            program.id,
+            /* search= */ Optional.empty(),
+            /* page= */ Optional.of(1), // Needed to skip redirect.
+            /* fromDate= */ Optional.empty(),
+            /* untilDate= */ Optional.empty(),
+            /* applicationStatus= */ Optional.empty(),
+            /* selectedApplicationUri= */ Optional.empty(),
+            /* showDownloadModal= */ Optional.empty(),
+            /* message= */ Optional.empty());
+    assertThat(result.status()).isEqualTo(OK);
+    // check if the page size is 100
+    assertThat(contentAsString(result)).contains("100");
```

Comment:
Question (blocking): Is it possible to assert on something more specific than "100"? (Or will this be better tested in #9017?)

Reply:
Good catch - yes asserting the status update form is present (checking by the form id)